WalletWasabi/Announcements/ReleaseHighlights.md

```diff
@@ -1,49 +1,36 @@
 
-However, maintaining and improving the software requires ongoing effort, such as maintenance, continuous improvement, bug fixes, testing, documentation, regular updates etc...
+We want to thank you for believing in Wasabi and for your support, whether through using the software, providing feedback, contributing financially, or contributing to the open-source project in any form.
 
-While Wasabi is one of the best tools available to use Bitcoin privately, achieving flawless financial-sovereign privacy is still a work in progress.
-There are many features yet to be implemented, and your support can help make them a reality.
+To show our gratitude, we've designed this release for you, our community. We've finally addressed some of the oldest bugs and feature requests that have been long-awaited!
 
-Additionally, contributors cover static costs, such as server expenses and release management, using their personal funds.
+Listing all the improvements would be too lengthy, but here are some examples:
+- More & better-designed settings (specifically for Coinjoin)
+- Randomly skipping rounds was disabled
+- Change avoidance suggestions are now more precise
 
-A donate button has been added on the main screen.<br/>
-It uses Silent Payments, so it's a great way to test that new feature!
+We expect these improvements to significantly enhance the overall experience of using Wasabi as your daily Bitcoin wallet.
 
-By donating, you’re directly supporting the developers who work tirelessly to maintain and improve Wasabi Wallet and make Bitcoin privacy accessible for everyone.
-
-#### 🧪 ~~TestNet 3~~ -> TestNet 4
-
-Instead of TestNet 3, Wasabi now uses TestNet 4.
-
-This new network has slightly different mechanisms that mitigates some of the problems that were undermining TestNet 3 to the point that it is basically unusable.
-
-While using the test network will be much more enjoyable now, coins from TestNet 3 don't exist on this new network. You must use a faucet and get yourself TestNet 4 coins.
-
-### 📝 Release Notes in the client
-
-If you've reached this far, you've probably noticed that the release notes highlights are now available in the client! They are displayed every first launch after an update.
-
-Full release notes are still available on GitHub.
+### 📦 Backend and Coordinator packaged for Linux
 
+The Debian package now includes two extra binaries: one for the backend (Wallet API) and one for the coordinator (Coinjoin API). This update simplifies the deployment process for Linux users, making it straightforward to set up their backend servers and coordinators.
 
+This ease of use lowers the barrier to contributing to the resiliency of Wasabi's infrastructure against potential attacks and technical failures. It also makes it possible to create private Wasabi infrastructures with private coordinators and backends.
```

Comment:
> This ease of use lowers the barrier to contributing to the resiliency of Wasabi's infrastructure against potential attacks and technical failures

maybe simplify with something like: makes it easier to run a backend and/or coordinator

> It also makes it possible to create private Wasabi infrastructures with private coordinators and backends.

this is not so clear imo, what do you mean to say

Reply:
> what do you mean to say

I mean that you can run a backend/coordinator for your community, company, club or whatever. Imagine your company hosts a coordinator and all employees can access to the vlan and schedule payments in coinjoins.